gui/job-details.lua

```diff
@@ -346,24 +367,44 @@ local function ScrJobDetails()
     return df.global.game.main_interface.job_details
 end
 
+local function ScrWorkorderConditions()
+    return df.global.game.main_interface.info.work_orders.conditions
```

Comment:
this is statically allocated and can be cached. e.g.
local conditions = df.global.game.main_interface.info.work_orders.conditions

Reply:
job_details above is static as well